repos/system_upgrade/el7toel8/actors/satellite_upgrade_data_migration/actor.py

```diff
@@ -38,11 +38,10 @@ def process(self):
                 os.unlink(service)
 
         if facts.postgresql.local_postgresql and os.path.exists(POSTGRESQL_SCL_DATA_PATH):
-            # remove empty PostgreSQL data from the package
-            if os.path.exists(POSTGRESQL_DATA_PATH):
-                os.rmdir(POSTGRESQL_DATA_PATH)
+            # we can assume POSTGRESQL_DATA_PATH exists and is empty
```

Comment:
Can we? What makes it so that we can assume it? A pre check?

Reply:
```console
# rpm -qf /var/lib/pgsql/data/
postgresql-server-12.11-2.module_el8.6.0+1153+eb826827.x86_64
```

And we install the package.

As for empty:
https://github.com/oamg/leapp-repository/blob/65fec1995d791cee8cf59e4b2ec1cbccdc4e8757/repos/system_upgrade/el7toel8/actors/satellite_upgrade_check/libraries/satellite_upgrade_check.py#L8-L21

Reply:
Doesn't that sort of guarantee it does NOT exist? I'd expect a common solution for users will be `rm -rf /var/lib/pgsql/data` as that would remove the inhibitor.

Reply:
That's a good point!

Reply:
Actually, no, this is still true.

If the user has an EL7 install that started with PostgreSQL 9.2 in `/var/lib/pgsql` and then was migrated to PostgreSQL 12 in `/var/opt/rh/rh-postgresql12/lib/pgsql` and the user plain `rm -rf /var/lib/pgsql` or `rm -rf /var/lib/pgsql/data` to clean up the old cruft, the installation of PostgreSQL 12 from the EL8 module will re-recreate all these folders for us.

Reply:
Not a 100% reproducer, but close enough to confirm that as long as we (re)install `postgresql-server`, the directory should be present:
```console
# rpm -qf /var/lib/pgsql/data
postgresql-server-12.11-2.module_el8.6.0+1153+eb826827.x86_64
# rm -rf /var/lib/pgsql/data
# dnf reinstall postgresql-server -y
# ls -d /var/lib/pgsql/data
/var/lib/pgsql/data
```